src/main/kotlin/com/papsign/ktor/openapigen/model/schema/SchemaModel.kt

```diff
@@ -60,6 +60,18 @@ sealed class SchemaModel<T>: DataModel {
         override var description: String? = null
     ) : SchemaModel<T>()
 
+    data class SchemaModelString(
+        var type: DataType? = null,
+        var format: DataFormat? = null,
+        var nullable: Boolean = false,
+        var minLength: Int? = null,
+        var maxLength: Int? = null,
+        var pattern: String? = null,
+        override var example: String? = null,
+        override var examples: List<String>? = null,
+        override var description: String? = null
+    ) : SchemaModel<String>()
+
```

Comment:
The properties should be added to `SchemaModelLitteral` as it is the model generated for strings

Reply:
Ok, I've removed the `SchemaModelString` class and added the new attributes to `SchemaModelLitteral`. The new attributes now appear in the swagger schemas, if the new annotations are used.